build.gradle

```diff
@@ -107,12 +107,6 @@ model {
       operatingSystem SupportedOS.valueOf(platform.toUpperCase())
       architecture "${arch}"
     }
-    if(platformArch != currentPlatformArch) {
-      create(currentPlatformArch) {
-        operatingSystem SupportedOS.CURRENT
-        architecture "${currentArch}"
-      }
-    }
```

Comment:
I got rid of this because it made the Gradle build linux i386 + x86_64 if you were only wanting to publish i386 but on x86_64 arch